java-client/src/test/java/org/opensearch/client/opensearch/experiments/ParsingTests.java

```diff
@@ -157,20 +157,21 @@ public void testSmartcn_StopFilter() {
         TokenFilterDefinition analyzer2 = fromJson(str, TokenFilterDefinition._DESERIALIZER);
     }
 
-    @Test
-    public void testCjk_Analyzer() {
-        final Analyzer analyzer = new Analyzer.Builder().cjk(b -> b.stopwords(Arrays.asList("a", "b", "c")).stopwordsPath("path")).build();
-
-        assertTrue(analyzer.isCjk());
-
-        String str = toJson(analyzer);
-        assertEquals("{\"type\":\"cjk\",\"stopwords\":[\"a\",\"b\",\"c\"],\"stopwords_path\":\"path\"}", str);
-
-        Analyzer analyzer2 = fromJson(str, Analyzer._DESERIALIZER);
-        assertTrue(analyzer2.isCjk());
-        assertEquals(analyzer.cjk().stopwords(), analyzer2.cjk().stopwords());
-        assertEquals(analyzer.cjk().stopwordsPath(), analyzer2.cjk().stopwordsPath());
-    }
+//   temporary commenting this test until we decide if we keep the CjkAnalyser
+//    @Test
+//    public void testCjk_Analyzer() {
+//        final Analyzer analyzer = new Analyzer.Builder().cjk(b -> b.stopwords(Arrays.asList("a", "b", "c")).stopwordsPath("path")).build();
+//
+//        assertTrue(analyzer.isCjk());
+//
+//        String str = toJson(analyzer);
+//        assertEquals("{\"type\":\"cjk\",\"stopwords\":[\"a\",\"b\",\"c\"],\"stopwords_path\":\"path\"}", str);
+//
+//        Analyzer analyzer2 = fromJson(str, Analyzer._DESERIALIZER);
+//        assertTrue(analyzer2.isCjk());
+//        assertEquals(analyzer.cjk().stopwords(), analyzer2.cjk().stopwords());
+//        assertEquals(analyzer.cjk().stopwordsPath(), analyzer2.cjk().stopwordsPath());
+//    }
 
```

Comment:
@Test
    public void testLanguage_Analyzer() {
        for (final Language language: Language.values()) {
	        final Analyzer analyzer = new Analyzer.Builder().language(b -> b.language(language).stopwords(Arrays.asList("a", "b", "c")).stopwordsPath("path")).build();
	
	        String str = toJson(analyzer);
	        assertEquals("{\"type\":\"" + language.jsonValue() + "\",\"stopwords\":[\"a\",\"b\",\"c\"],\"stopwords_path\":\"path\"}", str);
	
	        Analyzer analyzer2 = fromJson(str, Analyzer._DESERIALIZER);
	        assertEquals(analyzer.language().stopwords(), analyzer2.language().stopwords());
	        assertEquals(analyzer.language().stopwordsPath(), analyzer2.language().stopwordsPath());
	        assertEquals(analyzer.language().language(), language);
        }
    }
}